src/tv2-common/migrations/index.ts

```diff
@@ -171,15 +172,76 @@ export function mapGfxTemplateToDesignTemplateAndDeleteOriginals(
 	})
 }
 
+export function moveSelectedGfxSetupNameToGfxDefaults(versionStr: string) {
+	const fromValue = 'SelectedGfxSetupName'
+	const targetTable = 'GfxDefaults'
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.moveSelectedGfxSetupName.${fromValue}.toTable.${targetTable}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const singleValue = context.getBaseConfig(fromValue)
+			const designatedTable = context.getBaseConfig(targetTable)
+
+			if (!singleValue || !Array.isArray(designatedTable)) {
+				return false
+			}
+
+			return true
```

Comment:
This can be condensed to:
```typescript
return singleValue && Array.isArray(designatedTable)
```

Reply:
Unfortunately the validate method won't allow that. 
![image](https://user-images.githubusercontent.com/32238291/229476258-f56c8d2b-aadc-4702-b03c-368f72aac846.png)

Reply:
Makes sense, then either one of these:
```typescript
return singleValue !== undefined && Array.isArray(designatedTable)
// or
return Boolean(singleValue) && Array.isArray(designatedTable)
// or
return !!singleValue && Array.isArray(designatedTable)
```